targets/xorg

```diff
@@ -104,10 +104,9 @@ END
     apt-get -y --force-yes --no-install-recommends dist-upgrade -f
 fi
 
-install xorg arch=,xserver-xorg-video-fbdev$ltspackages
-if [ "${ARCH#arm}" = "$ARCH" ]; then
-    install arch=,xserver-xorg-video-intel$ltspackages
-fi
+install xorg arch=,xserver-xorg-video-fbdev$ltspackages arch=,xserver-xorg-video-intel$ltspackages
+# JMT: eventually wrap this removal with an appropriate test
+remove xf86-video-odroid-c1
```

Comment:
That's going to break non-ARM Chromebooks...

Instead of uninstalling stuff, I wonder if we could be a bit smarter about package selection, maybe install only `xorg-server`, `xorg-server-utils` and `xorg-apps`, and required drivers instead of the whole `xorg` group?